modules/consensus/consensusset.go

```diff
@@ -50,6 +48,11 @@ type ConsensusSet struct {
 	// up loading blocks from memory.
 	verificationRigor verificationRigor
 
+	// updatePath is a flag that determines if a block node will
+	// be added to the path. It should be false when loading the
+	// current path from disk and true otherwise
+	updatePath bool
```

Comment:
Is this a temporary variable? The current path will not need to be loaded into memory once everything has been moved to a database, right?